ftl/python/layer_builder.py

```diff
@@ -214,6 +214,93 @@ def _log_cache_result(self, hit):
                 key=self.GetCacheKey()))
 
 
+class PipfileLayerBuilder(RequirementsLayerBuilder):
```

Comment:
nit: there are now four separate `LayerBuilder` implementations in here, we should consider splitting them into separate files for clarity

Reply:
def agree here